aiida_gaussian/calculations/gaussian.py

```diff
@@ -213,32 +212,10 @@ def prepare_for_submission(self, folder):
         return calcinfo
 
     @classmethod
-    def _render_input_string_from_params(cls, param_dict, pmg_structure):
-        """
-        Generate the Gaussian input file using pymatgen
-
-        In the following, the param_dict.get() method is used, which returns
-        `None` if the corresponding key is not specified, resulting in the
-        default value defined by the pymatgen library. For example, charge
-        and spin_multiplicity are then defined based on the pymatgen molecule
-        definition.
-        """
-
-        inp = mgaus.GaussianInput(
-            pmg_structure,
-            title="input generated by the aiida-gaussian plugin",
-            charge=param_dict.get("charge"),
-            spin_multiplicity=param_dict.get(
-                "multiplicity", param_dict.get("spin_multiplicity")
-            ),
-            functional=param_dict.get("functional"),
-            basis_set=param_dict.get("basis_set"),
-            route_parameters=param_dict.get("route_parameters"),
-            input_parameters=param_dict.get("input_parameters"),
-            link0_parameters=param_dict.get("link0_parameters"),
-            dieze_tag=param_dict.get(
-                "dieze_tag", "#N"
-            ),  # normal print level by default
-        )
-
-        return inp.to_string(cart_coords=True)
+    def _render_input_string_from_params(cls, parameters, pmg_structure):
+        """Generate the Gaussian input file using pymatgen."""
+        parameters.setdefault("dieze_tag", "#N")
+        parameters.setdefault("spin_multiplicity", parameters.pop("multiplicity", None))
```

Comment:
@eimrek I kept this line in, but is this really necessary? Wouldn't it be easier to just document that users should use `spin_multiplicity` in the `parameters` instead of `multiplicity`? Or is `spin_multiplicity` really bad to use?

Reply:
I don't fully remember why we chose to prefer `multiplicity` over the default pymatgen `spin_multiplicity`, but I think at this stage, it's good to leave this in, for backwards compatibility. In principle, in the README, we could switch to `spin_multiplicity`, so that it more directly adheres to the pymatgen input.